tests/pagination/tests.py

```diff
@@ -233,6 +233,14 @@ def test_get_page_hook(self):
         self.assertEqual(page2.previous_page_number(), 1)
         self.assertIsNone(page2.next_page_number())
 
+    def test_page_range_iterator(self):
+        """
+        Tests that the Paginator is an iterator.
```

Comment:
when possible, I like to avoid "Tests that.." boilerplate since that's always the point of tests.
suggestion: `Paginator.page_range should be an interator.`